api/ssi-bridge/src/database/user.ts

```diff
@@ -49,12 +49,13 @@ export const addUser = async (user: UserPersistence) => {
 	if (user.verifiableCredentials?.length >= maxNumberOfVc) {
 		throw new Error(`You can't add more than ${maxNumberOfVc} verifiable credentials to a user!`);
 	}
+	const { role } = user;
 
 	const document = {
 		_id: user.id,
 		...user,
 		registrationDate: new Date(),
-		role: UserRoles.User
+		role: role ? role : UserRoles.User
```

Comment:
Using this everyone could make himself an admin. But I think it is not necessary for the root identity to be an admin since it has a credential to issue credentials to others. So we can remove this "requirement"